src/Compilers/VisualBasic/Portable/CommandLine/CommandLineDiagnosticFormatter.vb

```diff
@@ -78,35 +78,41 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 ' Builds leading spaces up to the error span
                 For position = Math.Min(sourceSpanStart, line.Start) To Math.Min(line.End, sourceSpanStart) - 1
+#Disable Warning RS0030 ' Do not used banned APIs - This isn't a parsing scenario.
```

Comment:
this just seems to be excessive.  we're talking about tiny amounts of time.  We don't need to ban a totally reasonable api in a blanket fashion like this.  We can simply use traces to drive things.

Reply:
If accessing 'Text' should be a problem there, then introduce a member that hides that, and have it either throw, or be obsolete so that statically you are stopped :)

Reply:
Fixed in af8bc88d3c650dda244d8dc4e214e6a4e9120b27